docs/source/algorithms/ComputeIncoherentDOS-v1.rst

```diff
@@ -0,0 +1,88 @@
+.. algorithm::
+
+.. summary::
+
+.. alias::
+
+.. properties::
+
+Description
+-----------
+
+Computes the phonon density of states from an inelastic neutron 
+scattering measurement of a powder or polycrystalline sample,
+assuming that all scattering is incoherent, using the formula 
+(Thermodynamic Properties of Solids, eds. Chaplot, Mittal, Choudhury,
+Chapter 3) for the 1-phonon incoherent scattering function:
+
+:math:`S^{(1)}_{\mathrm{inc}}(Q,E) = 
+   \exp\left(-2\bar{W}(Q)\right) \frac{Q^2}{E} 
+   \langle n+\frac{1}{2}\pm\frac{1}{2} \rangle
+   \left[ \sum_k \frac{\sigma_k^{\mathrm{scatt}}{2m_k} g_k(E) \right]`
```

Comment:
There is a problem with this latex formula.

Firstly it is not parsing as you have it split over multiple lines, which the ln line :math: command does not like.
However there is also a problem with the last frac term the number of brackets do not work.

This corrects it so it will parse, but I've no idea if that is the correct formula once if made my guessed corrections.
:math:`S^{(1)}_{\mathrm{inc}}(Q,E) =    \exp\left(-2\bar{W}(Q)\right) \frac{Q^2}{E}    \langle n+\frac{1}{2}\pm\frac{1}{2} \rangle   \left[ \sum_k \frac{\sigma_k^{\mathrm{scatt}}{2m_k}} {g_k(E)} \right]`

![image](https://cloud.githubusercontent.com/assets/1017173/14746995/1d56b014-08aa-11e6-81d1-8d219608a0c2.png)